CalibCalorimetry/CaloTPG/plugins/CaloTPGTranscoderULUTs.cc

```diff
@@ -60,10 +60,13 @@ class CaloTPGTranscoderULUTs : public edm::ESProducer {
   std::vector<int> ietah;
   std::vector<int> ZS;
   std::vector<int> LUTfactor;
+  bool linearLUTs_;
   double nominal_gain;
   double RCTLSB;
   int NCTScaleShift;
   int RCTScaleShift;  
+  double lsbQIE8;
+  double lsbQIE11;
```

Comment:
configuration parameters should be declared const (-> they have to be read in the initializer list of the constructor)
this applies also to the existing parameters